.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,62 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/java
+{
+	"name": "APIs Testing",
+	// Docker Image
+	"image": "mcr.microsoft.com/devcontainers/java:17-bullseye",
+	//"runArgs": ["--env-file",".env"],
+	"mounts": [],
+	// Available Features: https://containers.dev/features
+	"features": {
+		"ghcr.io/devcontainers/features/java:1": {
+			"version": "17",
+			"installMaven": "true",
+			"installGradle": "false"
+		},
+		"ghcr.io/devcontainers/features/docker-in-docker:2":{}, // Create child containers inside dev-container
+		"ghcr.io/devcontainers/features/sshd:1": {}, // Adds a SSH server into dev-container
+		"ghcr.io/devcontainers/features/node:1":{}, // intall nodejs
+		"ghcr.io/devcontainers-contrib/features/apt-get-packages:1": {
+			"packages": ["xdg-utils"] // allows integration with host browser for Power-app auth 
+		},
+		"ghcr.io/devcontainers/features/azure-cli:1":{}, // Install Azure CLI
```

Comment:
Are we using az cli here?
leave this commented if not needed for now. it makes the container spin-up way quicker.

Same may apply to other features. :)